test/integration/good/code-gen/cl.expected

```diff
@@ -568,56 +566,32 @@ class optimize_glm_model final : public model_base_crtp<optimize_glm_model> {
     
     try {
       Eigen::Matrix<local_scalar_t__, -1, 1> alpha_v;
-      alpha_v = Eigen::Matrix<local_scalar_t__, -1, 1>(k);
-      stan::math::fill(alpha_v, DUMMY_VAR__);
```

Comment:
Is this removed  because these are parameters and we know they will be read in immediately?

The model is [here](https://github.com/stan-dev/stanc3/blob/58b05aaf3ae02441ca986fc0b280ce2075e6ec9b/test/integration/good/code-gen/optimize_glm.stan#L27)

I guess there is never a case where we generate code for parameters without reading in right?

Reply:
Yeah so parameters this optimization will always happen since they are immediately read

Reply:
Cool cool. Resolved.